manticore/platforms/evm.py

```diff
@@ -249,8 +249,11 @@ def __init__(self, opcode, name, operand_size, pops, pushes, fee, description, o
     def parse_operand(self, buf):
         operand = 0
         for _ in range(self.operand_size):
-            operand <<= 8
-            operand |= ord(next(buf))
+            try:
+                operand <<= 8
+                operand |= ord(next(buf))
+            except:
+                raise TerminateState("Operand has insufficient bytes")
```

Comment:
We are currently using the disassembler in other projects that wont't have TerminateState. So we should handle this iterator exception upstream and leave StopIteration here.